koku/masu/processor/tasks.py

```diff
@@ -69,8 +69,9 @@
 REFRESH_MATERIALIZED_VIEWS_QUEUE = "reporting"
 REMOVE_EXPIRED_DATA_QUEUE = "remove_expired"
 SUMMARIZE_REPORTS_QUEUE = "process"
-UPDATE_COST_MODEL_COSTS_QUEUE = "reporting"
+UPDATE_COST_MODEL_COSTS_QUEUE = "priority_update"
```

Comment:
This value should not change. Not all cost-model update tasks are caused by a change in the UI. The tasks that are not caused by UI changes should not go in the priority queue.